lms/envs/common.py

```diff
@@ -157,6 +157,9 @@
     # Can be turned off if course lists need to be hidden. Effects views and templates.
     'COURSES_ARE_BROWSABLE': True,
 
+    # Can be turned off if course search input needs to be hidden. Effects templates.
+    'COURSES_ARE_SEARCHABLE': True,
```

Comment:
@martynjames Should this FEATURE flag be used in [SOL-776](https://openedx.atlassian.net/browse/SOL-776)? Seems like the right way to do it ... have one flag to manage the course discovery search in general